### PR TITLE
Updates on user profile tab

### PR DESCRIPTION
**Pull request by vepr-ua**
- Added css for input, textfield disabled attribute.
- Added user email to the profile tab, disabled input.

### CODE REVIEW

File: packages/thriver-accounts/lib/templates/profile.html [hunk 1]
```diff
@@ -1,6 +1,202 @@
...
+          </select>
+        </div>
+      </div>
+      <div class="form-group">
+        <label for="profile.emergencyTelephone" class="control-label">Emergency Contact Name</label>
+        <input id="profile.emergencyTelephone" type="text" name="profile.events.emergencyTelephone" class="form-control" value="{{ user.profile.events.emergencyTelephone }}">
+      </div>
+    </div>
+  </form>
+</template>
```
Comment (micchickenburger) [inline packages/thriver-accounts/lib/templates/profile.html]: Every file should end in a new line.

File: packages/thriver-accounts/lib/templates/profile.html [hunk 2]
```diff
@@ -1,6 +1,202 @@
...
-  {{> quickForm collection="Meteor.users" id="updateProfile" doc=doc type="update"
-    omitFields=omitFields fields="profile" validation="browser" 
-      buttonContent=submitValue}}
-</template>
+  <form id="profile">
+    <header>
+      <h2>{{heading}}</h2>
+    </header>
+    <div class="form-group-container">
+      {{ #each user.emails}}
```
Comment (micchickenburger) [inline packages/thriver-accounts/lib/templates/profile.html]: There seems to be a lot of inconsistencies surrounding bracket styles.  I think everywhere else in the codebase (though we have no linting for markup) we leave out spaces:  `{{something}}` except for templates:  `{{> template}}`.

File: packages/thriver-accounts/lib/client/profile.js [hunk 3]
```diff
@@ -1,15 +1,34 @@
...
+ * @method
+ *  @param {any} fieldName
+ *  @param {any} fieldValue
+ */
+const updateProfileField = (fieldName, fieldValue) => {
+  const profile = {};
+  profile[fieldName] = fieldValue;
+
+  Meteor.call('updateUserProfile', profile, (error) => {
+    if (error) console.log(error);
```
Comment (micchickenburger) [inline packages/thriver-accounts/lib/client/profile.js]: Any way to notify the user when there is an error?  You could throw an `aside` with classes `error-msg hide` at the top of the template, for example, that you could populate with the error message and unhide when there was an error.  At least notifying the user the update didn't work.

**Review by micchickenburger — CHANGES_REQUESTED** [review]
Looks great!  Just some stylistic changes requested, in  addition to adding user communication when there is an error.

File: packages/thriver-accounts/lib/client/profile.js (hunk 3 above)
Comment (vepr-ua) [reply packages/thriver-accounts/lib/client/profile.js]: okay i will take a look into it.

**Review by micchickenburger — APPROVED** [review]

**Pull request merged**